config/__init__.py

```diff
@@ -0,0 +1,22 @@
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
+
+match FLASK_ENV:
+    case "development":
+        from config.envs.development import DevelopmentConfig as Config  # noqa
+    case "dev":
+        pass
+    case "test":
+        from config.envs.test import TestConfig as Config  # noqa
+    case "production":
+        pass
```

Comment:
we need to call in DevConfig and ProductionConfig on the corresponding match cases